merge-build-push/merge-and-build.sh

```diff
@@ -15,7 +18,10 @@ OSE_VERSION="${MAJOR}.${MINOR}"
 if [ "${OSE_VERSION}" != "${OSE_MASTER}" ] ; then
   PUSH_EXTRA="--nolatest"
 fi
-BUILDPATH="${HOME}/go"
+
+# Use the directory relative to this Jenkins job.
+BUILDPATH="${WORKSPACE}/go"
+mkdir -p $BUILDPATH
 cd $BUILDPATH
 export GOPATH=`pwd`
```

Comment:
Because you are starting with a fresh workspace, you have to add the following after you export your GOPATH

`go get github.com/jteeuwen/go-bindata`